langchain4j-vertex-ai/src/main/java/dev/langchain4j/model/vertexai/internal/VertexAIDocument.java

```diff
@@ -0,0 +1,32 @@
+package dev.langchain4j.model.vertexai.internal;
+
+import com.google.gson.Gson;
+import dev.langchain4j.data.segment.TextSegment;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+/**
+ * This class is used to serialize the document to JSON.
+ */
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class VertexAIDocument {
+    private static final Gson GSON = new Gson();
+
+    private String memberOfIndex;
```

Comment:
is it needed for something?

Reply:
Yeap. This is what is saved and retrieve to return the segment with the result for a find

Reply:
I can't find where this field is accessed, could you please point me in the right direction?